src/main/resources/hudson/plugins/mavendeploymentlinker/MavenDeploymentProjectLinkerAction/jobMain.jelly

```diff
@@ -1,14 +1,24 @@
+<?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
     <j:if test="${it.hasLatestDeployments()}">
       <table style="margin-top: 1em; margin-left:1em;">
           <t:summary icon="package.gif">
           ${%latestDeployments}
-          ${it.latestDeployments.text}
+          <ul>
+          <j:forEach items="${it.latestDeployments.deployments}" var="deployment">
+              <li><a href="${deployment.url}">${deployment.name}</a></li>
```

Comment:
This does not disallow URLs with `javascript:` scheme, so I expect there's still an XSS vulnerability here.